lighthouse-cli/performance-experiment/report/injected-script.js

```diff
@@ -30,17 +30,39 @@ window.addEventListener('DOMContentLoaded', _ => {
   rerunButton.style.display = 'inline-block';
 
   rerunButton.addEventListener('click', () => {
-    rerunButton.setAttribute('status', 'running');
-    rerunLighthouse().then(() => {
+    rerunButton.setAttribute('data-status', 'running');
+    rerunLighthouse({blockedUrlPatterns: getUrlPatternsToBlock()}).then(() => {
       location.reload();
+    }).catch(err => {
+      rurunButton.setAttribute('data-status', 'stable');
```

Comment:
nit: `stable` is a little odd to me, since this button will change, probably fine for now.